api/management/commands/import_utils.py

```diff
@@ -889,10 +890,50 @@ def import_governismo():
         )
 
 
+def import_votacoes_sumarizadas():
+    """Carrega dados de votações sumarizadas"""
+
+    print_import_info("Votações Sumarizadas")
+
+    votacoes_sumarizadas_df = pd.read_csv("data/votos_sumarizados.csv")
```

Comment:
Ao importar os dados, me deparo com o seguinte erro:
`FileNotFoundError: File b'data/votos_sumarizados.csv' does not exist`

O nome do csv salvo pelo leggo-geral é [votacoes_sumarizadas.csv](https://github.com/parlametria/leggo-geral/blob/sprint-44/update_leggo_data.sh#L518)